### ISSUE TITLE
Let all types has the https://schema.org/additionalProperty

### ISSUE THREAD

**Issue by fishfree** [schemaorg/schemaorg#3540]
https://schema.org/additionalProperty supports arbitrary key-value pairs, so it's very suitable for extended description.
Or is it only neccessary to add this property in the Thing type? Then all other types can inherit it.

**Comment by MatthiasWiesmann** [comment]
Are you suggesting [additionalProperty](https://schema.org/additionalProperty) be a property of  `Thing`?

**Comment by fishfree** [comment]
@MatthiasWiesmann Yes, exactly. It will add great extensibility to describe things.

**Comment by MatthiasWiesmann** [comment]
I would bring this to the larger community, as this would be a significant change.

**Comment by smrgeoinfo** [comment]
Thing seem like a pretty obvious domain for additionalProperty....

**Comment by roblinksdata** [comment]
I've found that <https://validator.schema.org> does not complain about `schema:additionalProperty` on classes where it isn't defined, such as `schema:Dataset`.

I tested the following snippet in <https://validator.schema.org> and found that it passes validation, which seems curious to me since other non-schema-compliant properties appear to raise warnings or errors.

Is this expected behaviour?

```json
{
    "@context": "https://schema.org/",
    "@type": "Dataset",
    "name": "HOT: Niskin bottle samples",
    "description": "Something, Something, Something, Something, Something, Something, Something, Something, Something",
    "additionalProperty": {
        "@type": "PropertyValue",
        "propertyID": "https://dbpedia.org/page/Spatial_reference_system",
        "value": "https://www.w3.org/2003/01/geo/wgs84_pos"
    }
}
```

![image](https://github.com/user-attachments/assets/25cc3c07-b7ad-4fe5-bbe9-80c05dcad0ed)

**Comment by fthobe** [comment]
+1

**Comment by tahirawan077** [comment]
+1 
Also if its allowed on purpose whats the reason behind it? How is addidtionalProperty different from the others?

**Comment by plue** [comment]
Three worked cases for a narrower version of this proposal: `additionalProperty` on **`Organization`** specifically, rather than on every type.

We build a content system on a fixed profile of thirty schema.org types, and before writing it we mapped everything five small German organisations publish on their websites onto that profile, item by item — a sports club, a music school, a vocal ensemble, a tax firm and a trade guild, drawn from public lists rather than chosen. 98 items in total; the study was pre-registered for twenty houses and stopped at five, so these are counts, not rates. Where an item fit no declared property we walked a fixed ladder of escapes: `additionalType`, a value object, `additionalProperty`, an `ItemList`, a `DefinedTerm`, an `Action`, a `Role`.

The `additionalProperty` rung is where `Organization` differs from `Place`, `Product` and `Offer`: it is not domain-listed there, so for a fact *about the organisation itself* there is no key-value fallback at all. Facts of that kind that we could not place anywhere:

| House | The fact its website has to state | Why nothing declared on `Organization` carries it |
|---|---|---|
| Tax firm (Steuerberatungsgesellschaft) | The four named statutes it operates under — *Steuerberatungsgesetz*, its implementing regulation, the chamber's professional code, the fee regulation — a mandatory disclosure for every regulated profession in Germany | No property means *is bound by*; `memberOf` is the chamber, not the law |
| Trade guild (Innung, a public-law body of craft businesses) | It becomes a legal person only when the regional chamber of crafts approves its statutes — the guild's founding fact, published in § 1 of those statutes | Not a `foundingDate`, not an `identifier` (a guild has no register number), not a `memberOf` |
| Trade guild | *Joining us makes you a member of the state association and the national federation, and our member benefits hold only while we belong to them* — the rule its fee schedule bills for | A statement about what one membership implies; not a property of either organisation, and a string is the only honest carrier |

Two things worth saying against our own evidence. First, our initial reading blamed the vocabulary for more than this: we had recorded *"we are a member of the national association of music schools"* and *"our supervisory chamber is …"* as gaps, and both are simply `Organization.memberOf`, which we had overlooked; the indemnity insurer every German professional must name is carried by `hasCertification`. Four of our seven apparent gaps closed that way, against the released vocabulary, before we wrote this. Second, a key-value pair is the lossiest carrier there is — the guild's transitive-membership rule above is arguably not expressible by any property vocabulary, and `additionalProperty` would only give it a place to sit, not a meaning.

What is left is still the same shape every time: a regulated organisation has facts about *itself* that it is legally required to publish, and the type that describes it is one of the few with no escape hatch. That argues for `Organization` — a `LocalBusiness` already has it, being a `Place` too; a plain `Organization` does not — rather than for `Thing` — a smaller change with the mandatory-disclosure cases behind it.

For transparency: our profile now declares `additionalProperty` on `Organization` anyway, marked as an off-domain use, which is why we would rather it were not one.
